pyxdf/test/test_data.py

```diff
@@ -0,0 +1,68 @@
+from pathlib import Path
+from pyxdf import load_xdf
+import pytest
+import numpy as np
+
+
+# requires git clone https://github.com/xdf-modules/example-files.git
+# into the root xdf-python folder
+path = Path("example-files")
+extensions = ["*.xdf", "*.xdfz", "*.xdf.gz"]
+files = []
+for ext in extensions:
+    files.extend(path.glob(ext))
+files = [str(file) for file in files]
+
+
+@pytest.mark.parametrize("file", files)
+def test_load_file(file):
+    streams, header = load_xdf(file)
```

Comment:
When we have more files this could get unwieldy, but for now it's a good start.

Reply:
What exactly do you mean? This works with any number of test files because it's simply loading them all. If you meant the specific tests, yes, this will need to be refactored into separate tests sooner or later.

Reply:
It's 47 lines for a simple file and not feasible for longer files. For the first check if a file can be read correctly it's ok, but for other files it should test specific parts of the reader.